modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/application/mgt/ServiceProviderUserRoleValidationTestCase.java

```diff
@@ -0,0 +1,151 @@
+package org.wso2.identity.integration.test.application.mgt;
```

Comment:
let's add license header

Reply:
Fixed with https://github.com/wso2/product-is/pull/13308/commits/1d02ae0e5259cfa0048206403bf8503c61b05387.